scripts/ci/docker-compose/integration-redis.yml

```diff
@@ -21,7 +21,7 @@ services:
     image: redis:5.0.1
     volumes:
       - /dev/urandom:/dev/random   # Required to get non-blocking entropy source
-      - redis-db-volume:/data/presto
+      - redis-db-volume:/data/redis
```

Comment:
Is this related to adding Trino provider or just a fix on the way ?

Reply:
Just a fix on the way. tt has no real consequences other than redis data will be separated out during testing from presto data :)